services/grpc/grpc.js

```diff
@@ -26,6 +26,14 @@ class ZapGrpc extends EventEmitter {
     activeSubscriptions: {},
   }
 
+  static SUBSCRIPTIONS = [
```

Comment:
This has been removed in `next`. Looks like it's been re-added here by mistake.

Reply:
done